src/linker/utilities/docker_utils.py

```diff
@@ -52,19 +57,25 @@ def _run_container(
     input_data: List[Path],
     results_dir: Path,
     diagnostics_dir: Path,
+    config: Optional[Dict[str, str]],
```

Comment:
It seems like it might be cleaner to call this `environment` or something instead of `config`.

Reply:
I opted for config in this case b/c it's possible we use it for non-env configuration in the future.